cmd/security-secretstore-setup/res/configuration.toml

```diff
@@ -77,3 +77,7 @@ ConsulSecretsAdminTokenPath = "/tmp/edgex/secrets/edgex-consul/admin/token.json"
   Service = "appservice"
   Username = "appservice"
 
+[KongAdmin]
+ConfigTemplatePath = "/res/kong-admin-config.template.yml"
+ConfigFilePath = "/tmp/kong/kong.yml"
+ConfigJWTPath = "/tmp/edgex/secrets/edgex-security-proxy-setup/jwt.txt"
```

Comment:
Do we care the `.txt` is a windows extension and other files we have used something like `kong-admin-jwt`

Reply:
agreed that we don't need the extension here

Reply:
Updated the file name to `kong-admin-jwt`.